components/Carousel/Carousel.js

```diff
@@ -11,4 +16,8 @@ let carousel = document.querySelector();
     4. Those buttons are gonna need some click handlers.
     5. Think of how you would animate this compoennt. Make the cards slide in and out, or fade. It's up to you!
     6. Have fun!
-*/
\ No newline at end of file
+*/
+let leftButton = document.querySelector('.left-button');
```

Comment:
`leftButton` and `rightButton` are part of the functionality of the Carousel. If you had multiple carousels, would you want to declare a `leftButton` and `rightButton` for each of them, or would you want that to be automated with the creation of your carousel class objects?